chore(curriculum): add responsive piano workshop (#56734)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-piano/612e83ec2eca1e370f830511.md

---
id: 612e83ec2eca1e370f830511
title: Step 6
challengeType: 0
dashedName: step-6
---

# --description--

> Add a `link` element inside your `head` element. Give it a `rel` attribute set to `stylesheet` and an `href` attribute set to `styles.css`.

Reviewer: ```suggestion
Add a `link` element inside your `head` element. Set its `rel` attribute to `stylesheet` and its `href` attribute to `styles.css`.
```
